projects/wave-core-new/src/lib/datatable/table/table.component.ts

```diff
@@ -46,6 +50,25 @@ export class DataTableComponent implements OnInit, AfterViewInit, OnDestroy, OnC
         } else {
             this.emptyTable();
         }
+
+        this.selectedLayerSubscription = this.projectService.getSelectedFeatureStream().subscribe((selection) => {
+            this.selectedFeature$.next(selection);
+
+            if (!!this.paginator) {
+                for (let i = 0; i < this.dataSource.data.length; i++) {
+                    const feature = this.dataSource.data[i];
+                    if (feature.ol_uid === selection.feature) {
```

Comment:
is this a feature or a id?

Reply:
it's an id. I initially had `feature` and `layer` as attributes of the `selection`, both Ids, but the layer matching is currently done via brute force (cf https://trello.com/c/kJ7YjkT2). Should i rename it to `feature_id`?